2023/YoctoSummit-2023.11/fdo/fido-device-onboarding.md

```diff
@@ -0,0 +1,516 @@
+class: center, middle, intro
+
+# **FIDO Device Onboarding**
+
+# Late-binding Provisioning & Tales from the Trenches of Bleeding Edge Tech
+
+### Yocto Summit 2023
+
+## Tymoteusz Burak
+
+<img src="/remark-templates/3mdeb-presentation-template/images/logo.png"
+  width="150px" style="margin-left:-20px">
+
+---
+
+# `whoami`
+
+.center[<img
+  src="/remark-templates/3mdeb-presentation-template/images/tymek_burak.png"
+  width="150px" style="marking-top:-50px">
+]
+
+.center[Tymoteusz Burak]
+.center[_Junior Embedded Systems Developer_]
+.right-column50[
+- 6 months in 3mdeb
+- Integration of functionalities and the creation of Operating Systems for
+embedded devices in Yocto
+- Working on my Bachelor's Degree in Automation and Robotics
+]
+.left-column50[
+- <a href="mailto:tymoteusz.burak@3mdeb.com">
+  <img
+    src="/remark-templates/3mdeb-presentation-template/images/email.png"
+    width="24px" style="margin-bottom:-5px; margin-left:-15px"/>
+    tymoteusz.burak@3mdeb.com
+  </a>
+- <a href="https://www.linkedin.com/in/tymoteusz-burak-a108252a0/">
+  <img
+    src="/remark-templates/3mdeb-presentation-template/images/linkedin.png"
+    width="24px" style="margin-bottom:-5px; margin-left:-15px"/>
+    linkedin.com/in/tymoteusz-burak-a108252a0
+  </a>
+]
+
+---
+
+# Who we are ?
+
+.center[
+  .image-15[![](/remark-templates/3mdeb-presentation-template/images/coreboot-1024x1024.png)]
+  .image-15[![](/remark-templates/3mdeb-presentation-template/images/uefi-1024x1024.png)]
+  .image-15[![](/remark-templates/3mdeb-presentation-template/images/lvfs.png)]
+  .image-15[![](/remark-templates/3mdeb-presentation-template/images/yocto.png)]
+]
+.center[.image-35[![](/remark-templates/3mdeb-presentation-template/images/openpower.svg)]]
+
+- coreboot licensed service providers since 2016 and leadership participants
+- UEFI Adopters since 2018
+- Yocto Participants and Embedded Linux experts since 2019
+- Official consultants for Linux Foundation fwupd/LVFS project since 2020
+- IBM OpenPOWER Foundation members since 2020
+
+---
+
+# Agenda
+
+- What is FDO?
+- Existing implementations of FDO protocol
+- Challenges faced in integrating the
+[fido-device-onboard-rs](https://github.com/fdo-rs/fido-device-onboard-rs)
+project in Yocto
+    + Current rust implementations inside of Yocto
+    + Bitbake environment vs pkg-config
+- Demo presentation
+- Q&A
+
+---
+
+# What is FDO?
+
+<br>
+<br>
+
+.center[
+
+.left-column50[
+<img src="/img/fido-logo.jpeg" height="100px">
+]
+
+.right-column50[
+<img src="/img/intel_logo_2023.png" height="100px">
+]
+
+_An automatic onboarding protocol for IoT devices. Permits late binding of
+device credentials, so that one manufactured device may onboard,
+without modification, to many different IOT platforms_
```

Comment:
Unfortunately, that definition is very confusing. What is IoT device and IoT platform in this context?

Reply:
[Added more notes on this](https://github.com/3mdeb/conferences/blob/yocto_summit_2023_fdo/2023/YoctoSummit-2023.11/fdo/fido-device-onboarding.md?plain=1#L110C2-L116)

[Source is now quoted](https://github.com/3mdeb/conferences/blob/yocto_summit_2023_fdo/2023/YoctoSummit-2023.11/fdo/fido-device-onboarding.md?plain=1#L98C13-L98C13)